### ISSUE TITLE
Eleven packages not being ingested correctly

### ISSUE THREAD

**Issue by daveverwer** [SwiftPackageIndex/SwiftPackageIndex-Server#2949]
Running:

```sql
SELECT
	'https://swiftpackageindex.com/' || r.owner || '/' || r.name as spi_url,
	p.status,
	p.processing_stage,
	r.updated_at
FROM
	repositories r
	INNER JOIN packages p ON r.package_id = p.id
WHERE
	r.updated_at < '2024-03-17'
ORDER BY
	updated_at;
```

Shows 11 packages where there is something wrong:

![Screenshot 2024-03-18 at 11 21 08@2x](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/5180/34645b97-0e3c-4921-8a60-1da32e5e7a38)

What's interesting is that 10 of them seem to have been broken on the 15th. Looking at our PR merges for Friday the 15th, the only things that went in were the build error reporting changes and some CI  changes, which should not have caused any issues.

Needs debugging locally.

**Comment by daveverwer** [comment]
Tuist ingestion failure is #2936

**Comment by daveverwer** [comment]
All `sindresorhus` packages are also fixed with #2936 as Sindre added Buy Me a Coffee:

https://github.com/sindresorhus/.github/blob/db519b615e0ecde1f6526988fc15603603ef91ca/funding.yml#L3C1-L3C16

All should be fixed. I'll leave this open until tomorrow so I can double-check that all packages are being updated.

**Comment by daveverwer** [comment]
Checked this morning and no more packages are stuck 👍
